libs/sdk-core/src/reverseswap.rs

```diff
@@ -26,6 +27,16 @@ use tokio::time::{sleep, Duration};
 pub const ESTIMATED_CLAIM_TX_VSIZE: u64 = 138;
 pub const ESTIMATED_LOCKUP_TX_VSIZE: u64 = 153;
 
+impl SdkError {
```

Comment:
NIT: might be better to keep all `impl SdkError`s in the same place, for a better overview?

Reply:
Done [4edbbbe](https://github.com/breez/breez-sdk/pull/469/commits/4edbbbee20fdca35aebe326f3048148e0f9c7b48)